.github/workflows/ci_cmake.yml

```diff
@@ -18,27 +18,27 @@ env:
   CTEST_NO_TESTS_ACTION: "error"
   CMAKE_BUILD_PARALLEL_LEVEL: 4
   CTEST_PARALLEL_LEVEL: 0
+  CMAKE_INSTALL_PREFIX: ~/libs
```

Comment:
In p4est we default the install to `local` relative to the local directory. Would it make sense to align this?